src/kadmin/server/ovsec_kadmd.c

```diff
@@ -173,6 +173,10 @@ setup_loop(int proponly, verto_ctx **ctx_out)
                                    krb5_iprop_prog_1);
         if (ret)
             return ret;
+    } else if (proponly) {
+        fprintf(stderr, _("%s: add 'iprop_enable = true' to kdc realm "
+                          "configuration, when using '-proponly' switch\n"),
```

Comment:
I don't think this is quite what you mean.  The thing the error should be capturing is the conflict between `iprop_enable = false` (which happens to be the default) and the user explicitly requesting `-proponly`.